### DISCUSSION TITLE
Non-standard links are converted to normal text

### DISCUSSION THREAD

**Discussion by realmazharhussain** [community/community#23110] [New to GitHub]
<p>Hi,</p>
<p>I need the following link in my README.md and repository’s wiki</p>
<pre><code class="lang-markdown">[Show in App Store](appstream://io.github.realmazharhussain.GdmSettings)
</code></pre>
<p>But it shows up as normal text. How do I create such link?</p>

**Comment by queenofcorgis** [comment]
<p>Hi there - first thing to check is that the files that you’re using have the .md extension - can you confirm that please?</p>

**Comment by jfullstackdev** [comment]
<p>it is not just recognized as a URL, is that the complete URL ? because if I will be adding <code>https://</code>, which is NOT correct, just to test it, it will be a link</p>
<p><code>[Show in App Store](https://appstream://io.github.realmazharhussain.GdmSettings)</code></p>
<p>like this</p>
<p><img src="https://user-images.githubusercontent.com/47092464/181074899-dd54d74e-1e26-4879-b50e-a324529b9dd8.png" alt="image" data-base62-sha1="twxBOpoMg3xDToTc6pxSmWbsag3" width="345" height="130"></p>

**Comment by realmazharhussain** [comment]
<aside class="quote no-group quote-modified" data-username="jdevstatic" data-post="3" data-topic="257606" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 jdevstatic:</div>
<blockquote>
<p>it is not just recognized as a URL, is that the complete URL ? because if I will be adding <code>https://</code>, which is NOT correct, just to test it, it will be a link</p>
<p><code>[Show in App Store](https://appstream://io.github.realmazharhussain.GdmSettings)</code></p>
<p>like this</p>
<p>[Not allowed to embed image here]</p>
</blockquote>
</aside>
<p>Is it a complete URL? Yes.</p>
<p><code>appstream://</code> is not an internet protocol. On Linux systems with a graphical store installed, opening <code>appstream://app_id</code> links opens up page for app with <code>app_id</code> in the graphical app store.</p>

**Comment by realmazharhussain** [comment]
<aside class="quote group-github-employees" data-username="liligalante" data-post="2" data-topic="257606" data-full="true">
<div class="title">
<div class="quote-controls"></div>
 liligalante:</div>
<blockquote>
<p>Hi there - first thing to check is that the files that you’re using have the .md extension - can you confirm that please?</p>
</blockquote>
</aside>
<p>Yes, it is indeed a markdown file.</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>yup, that’s why, and it’s not recognized,</p>
<p>just a quick test from <code>bitly</code>, it’s not recognized too</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/6/0/60b856a6036b3c089a7fd44792c57fce73840c77.png" data-download-href="/uploads/short-url/dNCMoiAysPfePsDyE4gFMPR1F5R.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074900-3673857e-9883-436f-8500-9b6a968e058e.png" alt="image" data-base62-sha1="dNCMoiAysPfePsDyE4gFMPR1F5R" width="690" height="122" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/6/0/60b856a6036b3c089a7fd44792c57fce73840c77_2_690x122.png, https://aws1.discourse-cdn.com/github/optimized/3X/6/0/60b856a6036b3c089a7fd44792c57fce73840c77_2_1035x183.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/6/0/60b856a6036b3c089a7fd44792c57fce73840c77.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/6/0/60b856a6036b3c089a7fd44792c57fce73840c77_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1094×194 14.6 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by queenofcorgis** [comment]
<p>Thanks for jumping in <a class="mention" href="/jdevstatic">@jdevstatic</a> :heart: ! I’ll let you take the reigns :horse_racing:t3:</p>
